README.md

```diff
@@ -25,42 +25,21 @@ and start contributing!
 
 *The API is in a very early stage. Everything is subject to change.*
 
-### /ec2 ###
-
-Search EC2 instances.
-
-Optional parameters:
-
-- `q`/`query`: OGNL expression used to filter (defaults to all instances). Examples:
-
-  - `state=="running"&&key=="pierre"&&launchTime>1365000000000`
-
-- `s`/`sort`: OGNL expression used as a `Comparable` to sort (default to no ordering).
-Example: `daysOld`.
-
-- `l`/`limit`: maximum number of records to return (defaults to none). Example: `10`.
-
-- `f`/`fields`: comma-separated list of fields to display (defaults to all). Examples:
-
-  - `type`
-  - `az,type`
-  - `id,publicIP,launchTime`
 
 ### /ec2/all ###
 Search all ec2 instances.
 
+### /ec2/sg ###
+Get ec2 security groups.
```

Comment:
What's an example of this? Why are we pulling sg out specifically?

We should keep the ec2 docs together. Organizationally this readme is hard to follow as it jumps around and not every end point supports queries. How we organize the readme layout should reflect that.